bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -40,6 +40,19 @@ import (
 	"strings"
 )
 
+// Builder defines the methods used to create KfApps.
+// Primary purpose is to allow injecting a fake for use in testing.
+type Builder interface {
+	LoadKfAppCfgFile(cfgFile string) (kftypes.KfApp, error)
```

Comment:
do you need create here? and below?

Reply:
I don't think so. This is sufficient right now for testing kfctlServer (part of click to deploy).
I can; will add more methods in the future if we need it.